.github/workflows/release.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
       # Check if demo version changed
       - name: Demo version check
         id: demoCheck
-        uses: MontyD/package-json-updated-action
+        uses: MontyD/package-json-updated-action@1.0.1
```

Comment:
Shouldn't this be "@v1.0.1" with a "v"?

Reply:
The GitHub Action marketplace defines it like that, but for other Actions we used the "v". I think it should work both ways. 
![image](https://user-images.githubusercontent.com/1659006/199996961-1169848d-e0f1-4ccd-b7f7-4b050114f742.png)
https://github.com/marketplace/actions/package-version-updated

Reply:
Ok! I had only seen the usage example in which there is no version